bootique-jetty/src/main/java/io/bootique/jetty/cors/BootiqueCorsFactory.java

```diff
@@ -0,0 +1,148 @@
+package io.bootique.jetty.cors;
+
+import io.bootique.annotation.BQConfigProperty;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import static org.eclipse.jetty.servlets.CrossOriginFilter.*;
+
+/**
+ * @since 0.26
+ */
+public class BootiqueCorsFactory {
```

Comment:
This is a factory that doesn't create any objects. It needs a factory method, instead of creating the object instead of getters for downstream code to read its properties.